servicetest/environment/test_environment.py

```diff
@@ -141,6 +143,59 @@ def create_testoutput_dir(scope="module"):
     }
 ]
 
+
+# TODO: Clean up and refactor below to be nice when in a working state
+environment_test_cap = Capability("environment.test.cap",
+                                  [
+                                      {"id": "env", "config": GW_CONFIG_ONE_VAR},
+                                      {"id": "env", "config": GW_CONFIG_SAME_VAR}
+                                  ])
+
+test_cap_2 = Capability("environment.test.cap.2",
+                        [
+                            {"id": "env", "config": GW_CONFIG_APPEND}
+                        ])
+
+"""
+"""
+test_cap_3 = Capability("environment.test.cap.3",
+                        [
+                            {"id": "env", "config": GW_CONFIG_TWO_VARS},
+                            {"id": "env", "config": GW_CONFIG_APPEND}
+                        ])
+
+test_cap_4 = Capability("environment.test.cap.4",
+                        [
+                            {"id": "env", "config": GW_CONFIG_ONE_VAR}
+                        ])
+
+test_cap_5 = Capability("environment.test.cap.5",
+                        [
+                            {"id": "env", "config": GW_CONFIG_TWO_VARS},
+                            {"id": "env", "config": GW_CONFIG_PREPEND}
+                        ])
+
+test_cap_6 = Capability("environment.test.cap.6",
+                        [
+                            {"id": "env", "config": GW_CONFIG_TWO_VARS}
+                        ])
+
+manifest = StandardManifest(TESTOUTPUT_DIR,
+                            "environment-test-manifest.json",
+                            [
+                                environment_test_cap,
+                                test_cap_2,
+                                test_cap_3,
+                                test_cap_4,
+                                test_cap_5,
+                                test_cap_6
+                            ])
+
+
+def service_manifests():
+    return [manifest]
```

Comment:
Would be nice to define manifest in this function scope, lowering the stuff in global scope.